### PR TITLE
Import Linux Kernel typos: letter S

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -10812,6 +10840,7 @@ siezed->seized, sized,
 siezing->seizing, sizing,
 siezure->seizure
 siezures->seizures
+sift->shift
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: `sift` is a word

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -10843,6 +10874,8 @@ signular->singular
 signularity->singularity
 silentely->silently
 silenty->silently
+sill->still
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: `sill` is a word

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -10941,6 +10982,7 @@ snyc->sync
 socail->social
 socalism->socialism
 socities->societies
+socked->socket
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: `socked` is a word

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: removed

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: removed

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: removed

**Pull request merged**